dvc/stage/imports.py

```diff
@@ -3,10 +3,20 @@
 logger = logging.getLogger(__name__)
 
 
-def update_import(stage, rev=None):
+def _update_import_on_remote(stage, remote, jobs):
+    url = stage.deps[0].path_info.url
+    stage.outs[0].hash_info = stage.repo.cloud.transfer(
+        url, jobs=jobs, remote=remote, command="update"
+    )
```

Comment:
Hm, seems like this is something we are doing over and over in add/import and now update. Makes me think this should really be some kind of flag/option for `stage.save()` and `out.save()` that would make it transfer stuff. We've had this discussion before and it wasn't clear back then, but maybe it is a good time. Or at least we could consider it and do it in the follow up.

Reply:
I also thought the same, and I think it would somehow make sense to have a function to do this in either stage or output.